app/views/individual/incometax/business/WhatYearToSignUp.scala.html

```diff
@@ -0,0 +1,98 @@
+}
+@optionCurrentContent = {
+    <p class="govuk-body"> @messages("business.what_year_to_sign_up.conditional.1.line_1.signup") </p>
+    <ul class="govuk-list">
+        <li>@messages("business.what_year_to_sign_up.quarterly.date_1", (endYearOfCurrentTaxPeriod - 1).toString)</li>
+        <li>@messages("business.what_year_to_sign_up.quarterly.date_2", (endYearOfCurrentTaxPeriod - 1).toString)</li>
+        <li>@messages("business.what_year_to_sign_up.quarterly.date_3", endYearOfCurrentTaxPeriod.toString)</li>
+        <li>@messages("business.what_year_to_sign_up.quarterly.date_4", endYearOfCurrentTaxPeriod.toString)</li>
+    </ul>
+    <p class="govuk-body"> @messages("business.what_year_to_sign_up.conditional.1.line_2.signup", (endYearOfCurrentTaxPeriod + 1).toString) </p>
+}
+@optionNextContent = {
+    <p class="govuk-body"> @messages("business.what_year_to_sign_up.conditional.2.line_1.signup") </p>
+    <ul class="govuk-list">
+        <li>@messages("business.what_year_to_sign_up.quarterly.date_1", endYearOfCurrentTaxPeriod.toString)</li>
+        <li>@messages("business.what_year_to_sign_up.quarterly.date_2", endYearOfCurrentTaxPeriod.toString)</li>
+        <li>@messages("business.what_year_to_sign_up.quarterly.date_3", (endYearOfCurrentTaxPeriod + 1).toString)</li>
+        <li>@messages("business.what_year_to_sign_up.quarterly.date_4", (endYearOfCurrentTaxPeriod + 1).toString)</li>
+    </ul>
+    <p class="govuk-body"> @messages("business.what_year_to_sign_up.conditional.2.line_2.signup", (endYearOfCurrentTaxPeriod + 2).toString) </p>
+}
+
+@inputContent = {
+    <p>@messages("business.what_year_to_sign_up.line_1.signup")</p>
+}
+
+@backLink = @{
+    if(isEditMode) {
+        Some(backUrl)
+    } else {
+        None
+    }
+}
+
+@mainTemplate(title = pageTitle, optForm = Some(accountingYearForm), backLink = backLink) {
+
+    @form(action = postAction) {
+        @radioHelper(
+            field = accountingYearForm(AccountingYearForm.accountingYear),
+            legend = question,
+            hint = Some(messages("business.what_year_to_sign_up.line_1.signup")),
+            radioItems = Seq(
+                RadioItem(
+                    content = Text(messages("business.what_year_to_sign_up.option_1.signup", (endYearOfCurrentTaxPeriod - 1).toString, endYearOfCurrentTaxPeriod.toString)),
+                    conditionalHtml = Some(optionCurrentContent),
+                    value = Some(Current.toString)
+                ),
+                RadioItem(
+                    content = Text(messages("business.what_year_to_sign_up.option_2.signup", (endYearOfCurrentTaxPeriod).toString, (endYearOfCurrentTaxPeriod + 1).toString)),
+                    conditionalHtml = Some(optionNextContent),
+                    value = Some(Next.toString)
+                )
+            )
+        )
+        @if(isEditMode) {
+            @continueButton(Some(messages("base.update")))
+        } else {
+            @continueButton()
+        }
```

Comment:
Continue button can be simplified
```suggestion
        @continueButton(isEditMode = isEditMode)
```